src/main/java/it/pagopa/transactions/commands/handlers/v1/TransactionActivateHandler.java

```diff
@@ -208,8 +215,16 @@ public Mono<Tuple2<Mono<BaseTransactionEvent<?>>, String>> handle(
                         .collectList()
                         .flatMap(
                                 paymentRequestInfos -> jwtTokenUtils
-                                        .generateToken(transactionId, null)
+                                        .generateToken(
+                                                ecommerceSigningKey,
+                                                jwtEcommerceValidityTimeInSeconds,
+                                                new Claims(transactionId, null, null)
+                                        )
                                         .map(generatedToken -> Tuples.of(generatedToken, paymentRequestInfos))
```

Comment:
Remove map and use only fold
```suggestion
                                      .fold(
                                                Mono::error,
                                                generatedToken -> Mono.just( Tuples.of(generatedToken, paymentRequestInfos)
                                        )
                                        
```